compiler/resolution/callInfo.cpp

```diff
@@ -21,8 +21,9 @@
 #include "callInfo.h"
 #include "expr.h"
 
-
-CallInfo::CallInfo(CallExpr* icall) : call(icall), scope(NULL) {
+CallInfo::CallInfo(CallExpr* icall, bool checkonly)
+  : call(icall), scope(NULL), name(NULL),
+    actuals(), actualNames(), badcall(false) {
```

Comment:
This is a little difficult to read for me, I think because in most places where we define these types of lists, we do one per line (i.e. expr.cpp, stmt.cpp, symbol.cpp, etc.).  Does this match the style of other constructors in the file?  If not, or if there aren't others, I'd be tempted to follow the pattern of the other files I mentioned.

Also, did we really miss initializing all of these?!  So disappointed . . .